.circleci/test-deploy.yml

```diff
@@ -35,6 +35,18 @@ jobs:
             export PATH="$PATH:~/bin"
             public-api-diff --help
 
+  test-install-maestro:
+    macos:
+      xcode: 16.4.0
+    resource_class: m4pro.medium
+    steps:
+      - checkout
+      - <orb-name>/install-maestro
+      - run:
+          name: Verify Maestro installation
+          command: |
+            maestro --version
```

Comment:
Question (still a bit new to these): if the maestro installation is already being verified in `src/scripts/install-maestro.sh`, why do we need to check it here? Is it because this sort of a unit tests to check that the `<orb-name>/install-maestro` works as expected?

Reply:
Makes sense! Thanks for explaining!